permissions/create_iam_writer_permissions.sql

```diff
@@ -3,5 +3,6 @@ GRANT USAGE ON SCHEMA billing TO iam_writer;
 GRANT SELECT, INSERT, DELETE ON audit.audit_events TO iam_writer;
 GRANT SELECT, INSERT, DELETE ON billing.billing_events TO iam_writer;
 GRANT SELECT, INSERT, DELETE ON billing.fraud_events TO iam_writer;
+GRANT SELECT, INSERT, UPDATE ON billing.billing_status TO iam_writer;
```

Comment:
We don't want this ... `iam_writer` is the role used by the recorder service, which should not be able to write to billing status.

Reply:
😞 Thanks. Removed.